play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
```

Comment:
Were you going to remove the "piercing" part of the damage from the medic abilities? I think it will help keep the players from being confused.